CONTRIBUTING.md

```diff
@@ -261,6 +261,12 @@ Documentation is written in Markdown and reStructuredText. In particular, the do
 conda env create -f docs/environment.yml
 ```
 
+Alternatively, you can install the documentation dependencies in an existing environment using the following command:
+
+```bash
+conda env update -n <ENVIRONMENT> -f docs\environment.yml
```

Comment:
```suggestion
conda env update -n <ENVIRONMENT> -f docs/environment.yml
```

Thanks @powersparks, can we change the path separator to be a standard POSIX one?

Reply:
@powersparks - can you make this small change? We're ready to merge this after the change is made.

Thanks again!